hook-worker/src/worker.rs

```diff
@@ -234,6 +235,10 @@ async fn process_webhook_job<W: WebhookJob>(
 
     match send_result {
         Ok(_) => {
+            let insert_to_complete_duration = Utc::now() - webhook_job.job().created_at;
```

Comment:
nit: Shouldn't we be taking the insert to _complete_ duration _after_ we call `complete()`?

Reply:
Otherwise I'd say this is more of a "insert to processed duration"